registries/_format/char.md

```diff
@@ -0,0 +1,21 @@
+---
+owner: baywet
+issue:
+description: A single character
```

Comment:
How is this useful? Surely someone would just say `"type": "string", "minLength": 1, "maxLength": 1`.

Reply:
it's useful to project the correct type when generating client or server side code. Or generating the description from the server side code. I remember coming across it with a couple of descriptions in the past. 
But again if we want to be prescriptive instead I'd understand.

Reply:
@baywet are you using this to distinguish between types like C++'s `char` vs `string`? Keep in mind that many languages have no such distinction so perhaps a bit more context in the explanation would be helpful.

Reply:
correct, and yes many languages don't have that distinction but for those that do, that added precision is nice added value. Now the compiler keeps things in check for them, instead of runtime validation rules. Also the memory use is lowered which is a negligeable gain in this context arguably.

Reply:
looking for resolution here so we can finalize the PR, are we ok with the char format or should I remove it at this point?

Reply:
There are a few instances of it floating around https://cs.github.com/?scopeName=All+repos&scope=&q=%2F%22format%22%3A+%22char%22%2F  Curiously, the majority of them seem to be in Microsoft API descriptions.

Reply:
I think I'd prefer for this PR to just describe formats that we know are supported by JSON Schema 2020-12 and OpenAPI v3.1, and leave the proposed creation of new formats to subsequent PRs.

Reply:
@karenetheridge yeah it would probably be a straightforward separate PR.  "Seen in the wild with reasonably consistent use" should be reason enough to add it, but it's not technically "missing" in the way the others are.  The `int16` that @ralfhandl brought up that is also not part of the OAS 3.0 or 3.1 spec could also be a quick separate PR.

Reply:
Things like `int16` could arguably added to the spec directly (3.1.1 or 3.2, whatever's being planned as an interim revision).

Reply:
Suggestion, could we use the version field on top of the page to drive whether it's part of the spec or not? Or maybe a separate field (boolean flag) and that information would be in the table to call out the difference between formats that were designed for and the formats that have been seen in the wild with decent use?

Reply:
moved to a separate PR, resolving